probatus/interpret/model_interpret.py

```diff
@@ -230,6 +262,11 @@ def plot(self, plot_type, target_columns=None, samples_index=None, **plot_kwargs
                 - `'dependence'`: Dependence plot for each feature
                 - `'sample'`: Explanation of a given sample in the test data
 
+            target_set (str, optional):
+                The set for which the plot should be generated, either `train` or `test` set. We recommend using test
+                set, because it is not biased by model training. The train set plots are mainly used to compare with the
```

Comment:
Coding for this PR looks good. My only comment is based on this docstring here.

Basically, a user would use `.plot(target_set='test'` to interpret the model. Another use case is detecting shifts in data distribution (by comparing feature importance with `.plot(target_set='test'`.

Why not make it easy for the user?
- `.plot()` gives feature importance on test
- `.plot_shifts()` gives feature importance shifts between train and test, in order to help detect shifts in data distribution.

This will keep the methods simpler (less parameters) as well, thus easier to comprehend/learn for the user.

Reply:
There are two reasons why I would keep it as `plot`
- Consistency, because other modules have only one plot method, with parameters allowing to tweak what is plotted for a given object
- Plotting Train shap values can also be used for other purposes than detecting the shift. I underlined one of them in the docstring.